joomla/joomla-cms
# [4.0] Group action toolbar buttons usernotes

### Summary of Changes
This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368). This time the usernotes.

### Testing Instructions
Go to the user notes in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a user note.
